src/stripe/stripe.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@nestjs/common';
+import Stripe from 'stripe';
+import ErrorsTypes from '../errors/errors.enum';
+import { GlobalException } from '../exceptions/global.exception';
+
+@Injectable()
+export class StripeService {
+  private readonly stripe: Stripe;
+
+  constructor() {
+    const apiVersion = process.env.YOUR_STRIPE_API_VERSION;
+    const stripeKey = process.env.YOUR_STRIPE_SECRET_KEY;
```

Comment:
Could you rename this env variable just 'STRIPE_API_VERSION' , 'STRIPE_SECRET_KEY'